### ISSUE TITLE
[v4]  sheetRef.current?.close() hides bottom sheet modal permanentaly on Android with Expo SDK 44

### ISSUE THREAD

**Issue by ghost** [gorhom/react-native-scroll-bottom-sheet#809]
# Bug

I am using latest version of library.

I recently updated expo to latest stable version(SDK 44). The following bug is now noticeable:

## Environment info
"@gorhom/bottom-sheet": "^4",
"react-native": "0.64.3",
"react-native-reanimated": "~2.3.1",
"react-native-gesture-handler": "~2.1.0",

## Steps To Reproduce

- When you call sheetRef.current?.close(), the bottom sheet modal closes and is hidden permanently.
- Then if you try to call sheetRef.current?.present(), the bottom sheet modal remains hidden.
- If you try to open any other bottom sheet modal using some other ref, then other bottom sheet modal will open.
- When you close any other bottom sheet modal using swipe down gesture, the previous bottom sheet modal that was issued present() will appear.
- To summarize, close() does not work as intended, it should unwind the stack but doesnt. Swipe down works as intended and closes bottom sheet modal.

**Comment by ghost** [comment]
@gorhom hi, can you look into above issue when you are free. I dont know how to satisfy the bot to keep the issue open.

**Comment by ghost** [comment]
Turns out I was not at latest version, that is, "@gorhom/bottom-sheet": "4.1.4". Upon doing so, fixed the issue, thanks.

**Comment by ghost** [comment]
Seems like there was a bug in v4.1.4 where close() closes all open bottom sheet modals. Upgrading to v4.1.5 fixed this.
